src/Microsoft.AspNetCore.Razor.Language/TagHelperBinderSyntaxTreePass.cs

```diff
@@ -345,7 +345,8 @@ public override void VisitTagHelperPrefixDirectiveSpan(TagHelperPrefixDirectiveC
                 TagHelperDirectiveType directiveType)
             {
                 directiveText = directiveText.Trim();
-                if (directiveText.StartsWith("\"", StringComparison.Ordinal) &&
+                if (directiveText.Length >= 2 &&
```

Comment:
Found an issue with `@page` but nothing else stuck out as a potential problem. Lot of code to go through so very good chance I may have missed something. Nothing blatantly obvious though.